docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,2465 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    adata.obsp["distances_all"] = distance_matrix(adata.obsm["X_pca"], adata.obsm["X_pca"])
</pre><p>Why can't the sparse matrix be converted into a dense one for visualization? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='82'/>

Reply:
<p>I think I was inprecise here, not the sparsity is the problem, rather that it has too many zeros - only 9/2500 entries per row/column are nonzeros! So the <code>plt.imshow(adata.obsp["distances"].A)</code> basically looks like all-zeros. Thats why I use the full distance matrix here.</p><p> </p><p>Thinking about that, I felt the "distance" matrix in the data is maybe not even worth mentioning&amp;explaining at all, so I now removed that part completely to avoid confusion and distraction. If we make a "tailored" example data just for this tutorial, I suggest we remove the fields "distances" and "connectivities" from it, and maybe pre-compute the full distance matrix.</p>